### ISSUE TITLE
Test flake: aws-ts-voting-app

### ISSUE THREAD

**Issue by swgillespie** [pulumi/examples#260]
Seen in CI:

```
[ umi/examples/aws-ts-voting-app ] Destroying ([secure]/p-it-travis-job-aws-ts-vot-204ddacd):
[ umi/examples/aws-ts-voting-app ]     pulumi:pulumi:Stack voting-app-p-it-travis-job-aws-ts-vot-204ddacd  flag provided but not defined: -tracing
[ umi/examples/aws-ts-voting-app ]     pulumi:pulumi:Stack voting-app-p-it-travis-job-aws-ts-vot-204ddacd  Usage of tf-provider-flags:
[ umi/examples/aws-ts-voting-app ]     pulumi:pulumi:Stack voting-app-p-it-travis-job-aws-ts-vot-204ddacd    -get-provider-info
[ umi/examples/aws-ts-voting-app ]     pulumi:pulumi:Stack voting-app-p-it-travis-job-aws-ts-vot-204ddacd      	dump provider info as JSON to stdout
[ umi/examples/aws-ts-voting-app ] 
[ umi/examples/aws-ts-voting-app ]  -  aws:ecs:Service voting-app-frontend deleting 
[ umi/examples/aws-ts-voting-app ]  -  aws:ecs:Service voting-app-cache deleting 
[ umi/examples/aws-ts-voting-app ] @ Destroying................................
[ umi/examples/aws-ts-voting-app ]  -  aws:ecs:Service voting-app-frontend deleted 
[ umi/examples/aws-ts-voting-app ] @ Destroying....
[ umi/examples/aws-ts-voting-app ]  -  aws:ecs:Service voting-app-cache deleting error: Plan apply failed: deleting urn:pulumi:p-it-travis-job-aws-ts-vot-204ddacd::voting-app::awsx:x:ecs:FargateService$aws:ecs/service:Service::voting-app-cache: timeout while waiting for state to become 'INACTIVE' (last state: 'DRAINING', timeout: 10m0s)
[ umi/examples/aws-ts-voting-app ]  -  aws:ecs:Service voting-app-cache **deleting failed** error: Plan apply failed: deleting urn:pulumi:p-it-travis-job-aws-ts-vot-204ddacd::voting-app::awsx:x:ecs:FargateService$aws:ecs/service:Service::voting-app-cache: timeout while waiting for state to become 'INACTIVE' (last state: 'DRAINING', timeout: 10m0s)
[ umi/examples/aws-ts-voting-app ]     pulumi:pulumi:Stack voting-app-p-it-travis-job-aws-ts-vot-204ddacd  4 messages
[ umi/examples/aws-ts-voting-app ]  
[ umi/examples/aws-ts-voting-app ] Diagnostics:
[ umi/examples/aws-ts-voting-app ]   aws:ecs:Service (voting-app-cache):
[ umi/examples/aws-ts-voting-app ]     error: Plan apply failed: deleting urn:pulumi:p-it-travis-job-aws-ts-vot-204ddacd::voting-app::awsx:x:ecs:FargateService$aws:ecs/service:Service::voting-app-cache: timeout while waiting for state to become 'INACTIVE' (last state: 'DRAINING', timeout: 10m0s)
[ umi/examples/aws-ts-voting-app ]  
[ umi/examples/aws-ts-voting-app ]   pulumi:pulumi:Stack (voting-app-p-it-travis-job-aws-ts-vot-204ddacd):
[ umi/examples/aws-ts-voting-app ]     flag provided but not defined: -tracing
[ umi/examples/aws-ts-voting-app ]     Usage of tf-provider-flags:
[ umi/examples/aws-ts-voting-app ]       -get-provider-info
[ umi/examples/aws-ts-voting-app ]         	dump provider info as JSON to stdout
[ umi/examples/aws-ts-voting-app ]  
[ umi/examples/aws-ts-voting-app ] Resources:
[ umi/examples/aws-ts-voting-app ]     - 1 deleted
[ umi/examples/aws-ts-voting-app ] 
[ umi/examples/aws-ts-voting-app ] Duration: 10m2s
[ umi/examples/aws-ts-voting-app ] 
[ umi/examples/aws-ts-voting-app ] Permalink: https://app.pulumi.com/[secure]/voting-app/p-it-travis-job-aws-ts-vot-204ddacd/updates/2
[ umi/examples/aws-ts-voting-app ] error: update failed
```

**Comment by stack72** [comment]
@ellismg / @lukehoban I have ran this a lot over the weekend and I cannot replicate it - I would suggest someone else tries it a few times and if not, then we should just close it out

**Comment by lukehoban** [comment]
If we really don't think this is reproing anymore, I'll happy to turn the test back on.

**Comment by stack72** [comment]
Ok, this is still flakey!! This is a great example of customTimeout usecase!

**Comment by stack72** [comment]
I am going to track the work to fix this in #337
